src/System.Data.DataSetExtensions/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      _netfx;
```

Comment:
why is there one here but not in eg src\System.DirectoryServices\ref\Configurations.props

Reply:
Mistake, I just deleted it. We should leave the ref built as netstandard so that the tests are build against nestandard reference assembly but run against netfx inbox assembly. This will ensure that identities are the same. If we add an API here, when running vs netfx it would fail with a MissingMemeberException